src/GSoft.AspNetCore.Authentication.ClientCredentialsGrant/GSoft.AspNetCore.Authentication.ClientCredentialsGrant.csproj

```diff
@@ -11,6 +11,8 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Authentication" Version="2.2.0" />
```

Comment:
[Microsoft.AspNetCore.Authentication](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication) is an old ASP.NET Core deprecated package that hasn't been updated since 2018. You can safely remove it.